### ISSUE TITLE
DynamicDialog does not block interaction with elements behind the mask

### ISSUE THREAD

**Issue by benrhughes** [primefaces/primeng#10882]
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
The bug occurs in the primeng showcase: https://www.primefaces.org/primeng/showcase/#/dynamicdialog

**Current behavior**
When a dialog is open, it is still possible to interact with elements behind the background mask

**Expected behavior**
Elements behind the mask should be be accessible

**Minimal reproduction of the problem with instructions**

1. Press the "SHOW" button in the dynamicdialog showcase page
2. attempt to interact with any elements behind the mask

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**
primeng showcase

* **Angular version:** 13.0

* **PrimeNG version:** 13.0

* **Browser:** [ Chrome latest | Firefox latesst  | IE edge latest ]
 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

* **Node (for AoT issues):** `node --version` =

**Comment by EliasDerHai** [comment]
Same with PrimeNg 12

**Comment by mirzace** [comment]
I confirm the bug and it is quite important to solve. This is not happening on version 12.2.1.

It is related to #10809 which is marked as fixed in version 13.0.0.rc.1 but apparently it's not.
